src/components/Admin/AdminSearchForm.jsx

```diff
@@ -0,0 +1,163 @@
+/* eslint-disable camelcase */
+import qs from 'query-string';
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { Form } from '@edx/paragon';
+
+import SearchBar from '../SearchBar';
+import { updateUrl } from '../../utils';
+
+class AdminSearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    const { location } = props;
+    const queryParams = qs.parse(location.search);
+    this.state = {
+      searchQuery: queryParams.search,
+      searchCourseQuery: queryParams.search_course,
```

Comment:
Ah I see this snake_casing must be from python right? In js we typically do cameCasing, and I thought in the other FE libs we use a function to convert from pythonic to js-ic ... would that apply here too? would that be too much work? IT would be nice in my opinion, but I would not fret much if that adds too much work... just a thought